### PR TITLE
[jorai] update BCP47 code and language name

### PR DESCRIPTION
**Pull request by LornaSIL**

### CODE REVIEW

**Comment by LornaSIL** [comment]
Fixes #259

File: release/j/jorai/source/jorai.kps [hunk 1]
```diff
@@ -83,7 +83,7 @@
       <ID>jorai</ID>
       <Version>1.0</Version>
```
Comment (darcywong00) [inline release/j/jorai/source/jorai.kps]: This might need a keyboard/package version bump (history.md too)?

Comment (darcywong00) [reply release/j/jorai/source/jorai.kps]: @mcdurdin said for minor changes we can bump to version 1.0.1

**Comment by darcywong00** [comment]
The language tag updates lgtm

On a separate issue, the OSK for `z` and `x` (both unshifted and shifted) doesn't matching the keyboard rules
```
+ [SHIFT K_X] > '̆'
+ [SHIFT K_Z] > '̂'

+ [K_X] > '̆'
+ [K_Z] > '̂'
```

**Pull request merged**